osu.Game/Rulesets/Judgements/Judgement.cs

```diff
@@ -35,7 +35,40 @@ public class Judgement
         /// <summary>
         /// The minimum <see cref="HitResult"/> that can be achieved - the inverse of <see cref="MaxResult"/>.
         /// </summary>
-        public HitResult MinResult
+        /// <remarks>
+        /// Defaults to a sane value for the given <see cref="MaxResult"/>. May be overridden to provide a supported custom value:
+        /// <list type="table">
+        ///   <listheader>
+        ///     <term>Valid <see cref="MaxResult"/>s</term>
```

Comment:
```suggestion
        ///     <term><see cref="MaxResult"/>s</term>
```

I *think* this reads better to me?